app/views/unattended/provisioning_templates/provision/kickstart_default.erb

```diff
@@ -384,7 +386,6 @@ The last post section halts Anaconda to prevent endless loop in case HTTP reques
 
 if test -f /tmp/foreman_built; then
   echo "calling home: build is done!"
-  <%= indent(2, skip1: true) { snippet('built', :variables => { :endpoint => 'built', :method => 'POST', :body_file => '/root/install.post.log' }) } -%>
```

Comment:
In the past it has been discussed to split the built into two parts. So you would have some point where Foreman is notified that installation has completed, but isn't considered fully built.

I think this is important, because AFAIK `/root/install.post.log` is part of the installer `/root`, not `/root` on the final filesystem. Though we do have a step on line 371 to copy, so perhaps I'm mistaken here.

Reply:
> In the past it has been discussed to split the built into two parts. So you would have some point where Foreman is notified that installation has completed, but isn't considered fully built.

Then the part in if can be used as an indicator of machine being installed and callback from the machine can be taken as a sign of the machine being fully built.
 
> I think this is important, because AFAIK `/root/install.post.log` is part of the installer `/root`, not `/root` on the final filesystem. Though we do have a step on line 371 to copy, so perhaps I'm mistaken here.

From what I have tested, `install.post.log` is indeed copied to the /root of the final system.